src/Mail/PostAddedEmail.php

```diff
@@ -31,7 +31,7 @@ public function __construct(Post $post)
     public function build()
     {
         return $this
-            ->subject('New Post Added to Discussion "'.$this->post->discussion->title.'"')
+            ->subject(__('New post added to discussion') . ' "'.$this->post->discussion->title.'"' )
```

Comment:
Personally because this is a package I would lean toward having a `firefly` language file so that way anyone can easily override them in their own app.

Reply:
They can be overridden like this too @tomirons

Reply:
How would you override this mailable ?

Reply:
@andreaselia for it to work this way you would need a `.json` file. For a package I would lean more toward a language key e.g. `firefly.mail.post-added.subject`.

@answeb we're talking about overriding the language string, not the mailable.

Reply:
Sounds good @tomirons

Reply:
About translations, you don't even need to provide the `json` file, the user can update his `resources/lang/XX.json` to add the translations.

All your blades are localized, but you don't provide the language files ;)

Of course, the best would be to provide the language file, containing every localized text also from the blade templates.

Reply:
That's true, everywhere else uses the inline standard of locales so we can just do the same here.